dpdk.spec

```diff
@@ -81,6 +81,8 @@ ExclusiveArch: x86_64 aarch64 ppc64le
 %define incdir  %{_includedir}/%{name}
 %define pmddir %{_libdir}/%{name}-pmds
 
+%undefine _missing_build_ids_terminate_build
```

Comment:
Why is this necessary.  It's less-than-ideal to have to add it.

Reply:
some of the pmd build IDs are not generated and that terminates to build